### PR TITLE
Add AlbaSpot helm chart for OpenNESS

### PR DESCRIPTION
**Pull request by albora-tech**
This patch adds the helm chart for the AlbaSpot
application from Albora Technologies.

Signed-off-by: Ernest Artiaga <code@albora.io>

### CODE REVIEW

File: applications/albaspot/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+that provides continuous high-accuracy positioning. Its features include:
+
+* High-accuracy: centimetre-level accuracy in real time
+* Multi-constellation: supports GPS, Galileo, Beidou aand Glonass
+* Multi-frequency: supports L1, L2 and L5
+* Customizable: processing can be ported across edge, cloud and private data centers
+* Standardized: compatible with standard communication protocols (NTRIP/RTCM)
+* Fast and reliable: low latency due to optimized network design
+* Hardware agnostic: supports all third-party NTRIP/RTCM-compatible devices in the market
+
```
Comment (archie951) [inline applications/albaspot/README.md:35]: Hi,
Could you please add this section-  https://intel-https://github.com/open-ness/edgeapps/blob/master/DEVELOPING.md#pre-requisites---installing-openness. If I remember clearly OpenNESS version used is 21.03, flavor used is single node minimal and Distribution is OpenSource.

Comment (albora-tech) [reply applications/albaspot/README.md:35]: Done, additional information added.

Comment (archie951) [reply applications/albaspot/README.md:35]: Thanks. I will have to request you to please remove the IP from yaml file. You can mark it as xxx.xx.xx.xx if you want.

Comment (albora-tech) [reply applications/albaspot/README.md:35]: Thanks for the comment! I've removed the IP addresses as requested.

**Review by abjyoti — APPROVED** [review]
I am fine..

**Pull request merged**